tools/bazar/presentation/javascripts/components/BazarSearch.js

```diff
@@ -0,0 +1,53 @@
+// TODO better list and translatable
+var wordsToExcludeFromSearch = ['le', 'la', 'les', 'du', 'en', 'un', 'une']
+
+export default {
+  methods: {
+    searchEntries(entries, search) {
+      switch (this.params.search) {
+        case "dynamic":
+          return this.localSearch(entries, search)
+        case "true":
+          return this.distantSearch(entries, search)
+        default:
+          return entries
+      }
+    },
+    // Search throught API
+    distantSearch(entries, search) {
+      let params = { ...this.params, ...{ q: search } }
+      $.getJSON(wiki.url('?api/entries/bazarlist'), params, (data) => {
+        let searchedIds = data.entries.map(entry => entry[0])
```

Comment:
```suggestion
        let searchedIds = data.entries.map(entry => entry['id_fiche'])
```
par souci de stabilité, je pense que c'est mieux de récupérer `entry['id_fiche']` que `entry[0]` même si actuellement, ça représente la même chose, `id_fiche` a le mérite d'être explicitement nommé

Reply:
non c'est bien [0] qu'il faut ici, car les données sont envoyées sous forme de tableau pour gagner en place